Missionframework/modules/99_adminMenu/fnc/fn_adm_importSave.sqf

```diff
@@ -17,13 +17,13 @@
     BOOL
 */
 
-if (isServer) then {
-    // Toggle import switch for save skipping
-    KPLIB_save_imported = true;
+// Interrupt all mission functionalities, especially the saving
+KPLIB_campaignRunning = false;
+publicVariable "KPLIB_campaignRunning";
```

Comment:
I think this should be in `KPLIB_fnc_adm_importSaveServer`. This function should "delegate" what has to be done and the second one should do the job.

Right now if you will call `KPLIB_fnc_adm_importSaveServer` from for e.g. console it will not stop the campaign and bugs may occur :bug:

Reply:
Yeah, sounds sensible. 👍